### DISCUSSION TITLE
Does anyone know how to get the block placement position in PlayerBlockPlaceEvent?

### DISCUSSION THREAD

**Discussion by MasterBroNetwork** [Minestom/Minestom#1905] [Help]
Does anyone know if it's possible to get the position that the player is placing a block at in the PlayerBlockPlaceEvent?
I've tried event.getBlockPosition() but that just gets the position of the block that it is being placed on and not where the block that the player is trying to place is meant to go.

**Comment by KrystilizeNevaDies** [comment]
Are you sure that getBlockPosition is returning the block that the player right clicks on? It should be returning the block.
If it really isn't, you should be able to calculate it using getBlockFace instead.

**Reply by MasterBroNetwork** [reply]
From testing, It just keeps getting the position of the block that the player is right clicking on instead of where the block itself is being placed.
**Edit:** I'll try using getBlockFace to calculate it, Thanks for the tip.
